pkg/diagnostics/global_monitoring.go

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
@filintod  - I don't particularly like this or think this is 'the way'.  I just moved this out to get it separate for now.  Might be best for you our someone more close to the code to take over or otherwise make this look how you prefer.

Reply:
understood, but relying on import order is not robust enough. another idea could be to have a single Init for the package that calls all the individual xInits, no need to delay this PR more about it though. could be a future refactor PR. thanks for the feedback.

Reply:
you know actually we already have that setup this way in https://github.com/dapr/dapr/blob/a04348b0099df1890a88627333d0b1ecb3534e70/pkg/diagnostics/metrics.go#L48. We could just call this globa init in this InitMetrics. 

this is the better place to call instead of each individual xxx.Init(), sorry got confused when it was mentioned about import order

Reply:
I can see that each unit tests would need to call InitGlobals if moved out of each metric.Init, but that makes more sense that having it in each main path or in one that depends on certain order.  Another option, is to add an `init` method in one of the _test files to run this InitGlobals method in all the unit tests.